components/eam/bld/namelist_files/namelist_defaults_cam.xml

```diff
@@ -259,6 +259,7 @@
 <bnd_topo hgrid="ne0np4_northamericax4v1"  >atm/cam/topo/USGS_northamericax4v1_12xdel2_consistentSGH_191023.nc</bnd_topo>
 <bnd_topo hgrid="ne0np4_northamericax4v1" npg="2">atm/cam/topo/USGS_northamericax4v1pg2_12xdel2_consistentSGH_20020209.nc</bnd_topo>
 <bnd_topo hgrid="ne0np4_antarcticax4v1"  >atm/cam/topo/USGS-gtopo30_antarcticax4v1_12xdel2_consistentSGH_191120.nc</bnd_topo>
+<bnd_topo hgrid="ne0np4_antarcticax4v1"  npg="2">atm/cam/topo/USGS_gtopo30_aantarcticax4v1pg2_12xdel2_consistentSGH_20200925.nc</bnd_topo>
```

Comment:
`aantarctica` is misspelled here.

Reply:
antarctica is also misspelled on the title of the PR, but in a different way.

Reply:
Thanks for the sharp eyes, @ambrad , @maltrud . Corrected. Double checked the file in inputdata server is good.